<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1499</width>
    <height>998</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Acquisition setup&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="3" rowspan="2">
          <widget class="QDoubleSpinBox" name="_SB_tableFeed">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Pitch factor: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="_L_pitchFactor">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="3">
          <widget class="QDoubleSpinBox" name="_SB_initGantryAngle">
           <property name="suffix">
            <string> Â°</string>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="4" rowspan="2">
          <widget class="QPushButton" name="_PB_fitToVolumeSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>|&lt;--&gt;|</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Number of rotations:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="_SB_nbViews">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QDoubleSpinBox" name="_SB_initTablePos">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-9999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="_SB_nbRotations">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QPushButton" name="_PB_centerTrajectory">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>-&gt;|&lt;-</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Initial gantry angle [deg]:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number of views:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Initial table position [mm]:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="8">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1" rowspan="2" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Table feed per view [mm]:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Parameters</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="CTL::gui::AcquisitionVisualizerWidget" name="_W_geometryPreview" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Simulation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="_SB_nbPhotons">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_SB_focalSpotX">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_SB_focalSpotY">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QCheckBox" name="_CB_simulatePoisson">
           <property name="text">
            <string>Simulate Poisson noise</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Focal spot size:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Mean number of photons per pixel:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Options</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QCheckBox" name="_CB_simulateFocalSpot">
           <property name="text">
            <string>Simulate extended focal spot size</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="6">
          <widget class="QPushButton" name="_PB_simulateScan">
           <property name="text">
            <string>Run simulation</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Projection data&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2" colspan="2">
        <widget class="CTL::gui::ProjectionViewer" name="_W_projectionViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="_PB_saveProj">
         <property name="text">
          <string>Save projections</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Object data&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="1" colspan="2">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Values in:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLineEdit" name="_LE_loadFileName"/>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="_PB_loadPhantom">
           <property name="text">
            <string>Search file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Dimensions:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="4">
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>File</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="_SB_voxSizeX">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_SB_voxSizeY">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_SB_voxSizeZ">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Voxel size:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Âµ&lt;span style=&quot; vertical-align:sub;&quot;&gt;water&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_SB_muWater">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="suffix">
              <string> mmâ»Â¹</string>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="value">
              <double>0.027000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="_L_phantomDim">
           <property name="text">
            <string>0 x 0 x 0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="_RB_perMM">
             <property name="text">
              <string>&amp;1 / mm</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_RB_hounsfield">
             <property name="text">
              <string>H&amp;U</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="CTL::gui::VolumeViewer" name="_W_volumeView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="_statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CTL::gui::VolumeViewer</class>
   <extends>QWidget</extends>
   <header>gui/widgets/volumeviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CTL::gui::AcquisitionVisualizerWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/acquisitionvisualizerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CTL::gui::ProjectionViewer</class>
   <extends>QWidget</extends>
   <header>gui/widgets/projectionviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_SB_nbViews</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGeometryPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>1349</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_nbRotations</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGeometryPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>1350</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_tableFeed</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGeometryPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>1351</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_initGantryAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGeometryPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>793</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>1352</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_initTablePos</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGeometryPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>793</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>1354</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_nbViews</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updatePitchFactor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>1323</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_nbRotations</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updatePitchFactor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>1325</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_tableFeed</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updatePitchFactor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>1326</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_RB_perMM</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_22</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_RB_perMM</sender>
   <signal>toggled(bool)</signal>
   <receiver>_SB_muWater</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_RB_hounsfield</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_22</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_RB_hounsfield</sender>
   <signal>toggled(bool)</signal>
   <receiver>_SB_muWater</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateGeometryPreview()</slot>
  <slot>updatePitchFactor()</slot>
 </slots>
</ui>
